base/applications/mmc/console.c

```diff
@@ -152,16 +152,14 @@ FrameOnCreate(HWND hwnd,
     SetMenu(Info->hwnd,
             Info->hMenuConsoleSmall);
 
-    if (LoadAndFormatString(hAppInstance,
-                            IDS_APPTITLE,
-                            &lpTitle))
-	{
-		SetWindowText(Info->hwnd, lpTitle);		
-	}
-	else
-	{
-		SetWindowText(Info->hwnd, TEXT("ReactOS Management Console"));
-	}
+    if (AllocAndLoadString(&lpTitle, hAppInstance, IDS_APPTITLE))
+    {
+        SetWindowText(Info->hwnd, lpTitle);
+    }
+    else
+    {
+        SetWindowText(Info->hwnd, TEXT("ReactOS Management Console"));
```

Comment:
Note (to reviewer): do we (still) want such a hardcoded/default resource?

Reply:
(yes) how (else) would you (provide) a (fallback)?

Reply:
Let me know what you prefer, in order to get in finally approved